src/azure-web-command-service.ts

```diff
@@ -0,0 +1,77 @@
+import { Guid } from "guid-typescript";
+import { LocalStorageItem } from "./azure-web-local-storage-item";
+
+/**
+ * Manage local stored azure web event commands.
+ */
+export class AzureWebCommandService {
+    private static localStorageKey: string = "SENDMESSAGES";
+
+    /**
+     * Get a stored command by its id.
```

Comment:
Comments like these are nice, but not necessary IMO. It perfectly follows from the naming what it does already. These functions are public, but only used internally. Public exposed methods by the module are good for having comments though.

Reply:
When other developers work on this package they could use this public classes. Therefor i put comments above the public methods.